tests/suites/test_suite_hkdf.function

```diff
@@ -19,6 +19,10 @@ void test_hkdf( int md_alg, char *hex_ikm_string, char *hex_salt_string,
     unsigned char info[128] = { '\0' };
     unsigned char expected_okm[128] = { '\0' };
     unsigned char okm[128] = { '\0' };
+    /*
+     * okm_string is the string representation of okm,
+     * so its size is twice as the size of okm.
```

Comment:
Minor:
- `twice the size` instead of `twice as the size`.
- Suggestion: "hexadecimal string representation" instead of "string representation".
- Suggestion: Rename `okm_string` to `okm_hex`.